sdk/src/main/scala/com/horizen/account/utils/AccountFeePaymentsUtils.scala

```diff
@@ -49,7 +49,8 @@ object AccountFeePaymentsUtils {
         // sum all rewards for this forger address
         val forgerTotalFee = allForgersRewards
           .filter(info => forgerKey.equals(info.address))
-          .foldLeft(BigInteger.ZERO)((sum, info) => sum.add(info.value))
+          .map(_.value)
+          .reduce(_.add(_))
```

Comment:
does it work faster for us?
honestly, for me foldLeft approach was more clear.

Reply:
I would have preferred `map(_.value).sum`, but that doesn't work with the Java BigInteger, unfortunately. It would work with the Scala BigInt but we're not using that anywhere yet. Using `reduce` is still the better option I think, because it allows the List implementation to sum the elements in any order, potentially using a faster algorithm.

Reply:
done